src/styles/blocks/card.scss

```diff
@@ -0,0 +1,88 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+
+  &__image {
+    display: flex;
+    margin-bottom: 40px;
+
+    img {
+      height: 134px;
+      width: 160px;
+      margin: 0 auto;
+    }
+  }
+
+  &__name {
+    display: flex;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: $secondary-text-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 16px 0 24px;
+
+    .rating-reviews {
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+
+    .price-text {
+      font-size: 12px;
+      color: $secondary-text-color;
+    }
+
+    .price-value {
+      font-weight: 700;
+      font-size: 16px;
+    }
+  }
+
+  &__buy-button {
+    box-sizing: border-box;
+    text-transform: uppercase;
+    margin: 16px auto;
+    width: 166px;
+    height: 40px;
+    background-color: #00acdc;
+    padding: 0;
+    border: 0;
+    border-radius: 5px;
+    display: block;
+    color: $main-card-color;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    cursor: pointer;
+
+    &:hover {
+      border: 1px solid #00acdc;
+      color: #00acdc;
```

Comment:
Consider using variables for repeating colors

Reply:
Done!